### ISSUE TITLE
Microsoft Teams - "Sorry We Couldn't Connect You" when placing Call / Video session

### ISSUE THREAD

**Issue by JFish222** [meetfranz/franz#1758]
Hello,

I'm unable to make calls or join video sessions in Microsoft Teams.
Attempts result in "Sorry We Couldn't Connect You".

Attempted the same operation in Chrome Browser and calls/video sessions work perfectly.
I asked other members of my Team to give it a try and they report the same error when using Franz.

### Expected Behavior
Click the phone icon, and initiate voice call.

### Current Behavior
Attempts to initiate voice call result in described error.

### Steps to Reproduce (for bugs)
1.  Enter MS Teams session in Franz
2. Click on Team Channel or member in "chat" section.
3.  Click phone icon on upper right.
4.  Error occurs.  Same operations in web browser are successful.

### Context
We require the use of Video and Voice within Teams to hold our weekly calls/meetings.

### Your Environment
* Franz Version used: 5.4.0
* Operating System and version:  Win 10 1809

**Comment by dineshchand1984** [comment]
I am also facing the same issue. Let me know the fix if any.

**Comment by stonetwig** [comment]
I am also facing this issue. After being in a meeting I got kicked out of once I could rejoin and then I got kicked off again then this error started to occur. It feels like I was being actively blocked by microsoft from rejoining. There was no other issues with my connection and I have no firewall except a pihole dns server but I checked and it didn't block any requests. 

Tried to switch clients and downloaded the .deb file but with the same result. Switched to my laptop with the same result. Switched to my android after a download of the app with a different message but with the inability to rejoin the meeting. Even got off my wifi and just used the 4G connection, with the same result.

I got on my VPN connection and went on incognito, still the same result.

I use Ubuntu Linux on my desktop, Fedora Linux on my laptop and standard samsung android on my galaxy S9+.

---
Your Environment

Brave on Ubuntu, Brave on Fedora, latest Android
